src/prepare_data.py

```diff
@@ -0,0 +1,171 @@
+# author: Arlin Cherian, Kristin Bunyan, Michelle Wang, Berkay Bulut
+# date: 2021-11-24
+
+"""Cleans, preps, and feature engineers the data for the Coffee Quality Database dataset (https://github.com/rfordatascience/tidytuesday/blob/master/data/2020/2020-07-07/readme.md).
+Writes the training and test data to separate csv files.
+
+Usage: src/pre_process_coffee.py --input_data=<input_data> --out_dir=<out_dir>
+  
+Options:
+--input_data=<input_data>  Path (including filename) to downloaded data retrieved from download_data.py script 
+--out_dir=<out_dir>   Path to directory where the processed data should be written
+" -> doc
+"""
+import string
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+from docopt import docopt
+import os
+import pandas as pd
+
+opt = docopt(__doc__)
+
+def main(input_data, out_dir):
+    df = pd.read_csv(input_data, header=1)
+    
+    #drops the columns we are not using
+    df = df.drop(columns=['owner',
+                          'farm_name',
+                          'lot_number',
+                          'mill',
+                          'ico_number',
+                          'company',
+                          'altitude',
+                          'producer',
+                          'number_of_bags',
+                          'bag_weight',
+                          'in_country_partner',
+                          'certification_body',
+                          'certification_address',
+                          'certification_contact',
+                          'unit_of_measurement',
+                          'altitude_low_meters',
+                          'altitude_high_meters',
+                          'owner_1',
+                          'aroma',
+                          'clean_cup',
+                          'sweetness',
+                          'cupper_points',
+                          'flavor',
+                          'aftertaste',
+                          'body',
+                          'balance',
+                          'uniformity',
+                          'acidity',
+                          'region',
+                          'grading_date',
+                          'expiration'])
+        
+    #filter for Arabica
+    df = df.query('species == "Arabica"')
+    df = df.drop(columns=['species'])
+
+    #drop the null values
+    df = df.dropna()
+    
+    #function to clean region
```

Comment:
There is an indent, which should not be there, not sure if it will create any syntax errors.

Reply:
Those commands are within the main function, so it needs the indent at that point. Script runs on my machine with no reported syntax errors.